src/format/Kdbx3Writer.h

```diff
@@ -29,6 +29,8 @@ class Kdbx3Writer : public KdbxWriter
 
 public:
     bool writeDatabase(QIODevice* device, Database* db) override;
+
+    quint32 m_kdbxVersion = KeePass2::FILE_VERSION_3_1;
```

Comment:
What's the point of this? This should at least be a static constant, but I think it also makes sense to write the literal value in-place like before.